Error installing labbox-ephys-widgets-jp

Using the Windows Subsystem for Linux on Windows 10, running `pip install -e jupyterlab/labbox_ephys_widgets_jp` returns:
```Installing collected packages: labbox-ephys-widgets-jp
  Running setup.py develop for labbox-ephys-widgets-jp
    ERROR: Command errored out with exit status 1:
     command: /home/rly/miniconda3/envs/nwb_datajoint/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setup.py'"'"'; __file__='"'"'/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' develop --no-deps
         cwd: /home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/
    Complete output (90 lines):
    Code for extensions/ already in sync
    running develop
    running jsdeps
    > npm run build:all

    > labbox_ephys_widgets_jp@0.1.0 build:all
    > npm run build:labextension && npm run build:nbextension


    > labbox_ephys_widgets_jp@0.1.0 build:labextension
    > npm run clean:labextension && mkdirp labbox_ephys_widgets_jp/labextension && cd labbox_ephys_widgets_jp/labextension && npm pack ../..


    > labbox_ephys_widgets_jp@0.1.0 clean:labextension
    > rimraf labbox_ephys_widgets_jp/labextension


    > labbox_ephys_widgets_jp@0.1.0 prepack
    > npm run build:lib


    > labbox_ephys_widgets_jp@0.1.0 build:lib
    > tsc && npm run copy-files

    src/SortingViewPluginComponentWrapper.tsx(3,50): error TS2307: Cannot find module './extensions/common/getRecordingInfo'.
    src/pluginWidgets.tsx(15,103): error TS2724: Module '"./extensions/extensionInterface"' has no exported member 'sortingCurationReducer'. Did you mean 'sortingSelectionReducer'?
    npm ERR! code 1
    npm ERR! path /home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp
    npm ERR! command failed
    npm ERR! command sh -c tsc && npm run copy-files

    npm ERR! A complete log of this run can be found in:
    npm ERR!     /home/rly/.npm/_logs/2021-02-10T01_15_25_387Z-debug.log
    npm ERR! code 1
    npm ERR! path /home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp
    npm ERR! command failed
    npm ERR! command sh -c npm run build:lib

    npm ERR! A complete log of this run can be found in:
    npm ERR!     /home/rly/.npm/_logs/2021-02-10T01_15_25_416Z-debug.log
    npm ERR! code 1
    npm ERR! path /home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp
    npm ERR! command failed
    npm ERR! command sh -c npm run clean:labextension && mkdirp labbox_ephys_widgets_jp/labextension && cd labbox_ephys_widgets_jp/labextension && npm pack ../..

    npm ERR! A complete log of this run can be found in:
    npm ERR!     /home/rly/.npm/_logs/2021-02-10T01_15_25_431Z-debug.log
    npm notice
    npm notice New minor version of npm available! 7.0.14 -> 7.5.3
    npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.5.3>
    npm notice Run `npm install -g npm@7.5.3` to update!
    npm notice
    npm ERR! code 1
    npm ERR! path /home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp
    npm ERR! command failed
    npm ERR! command sh -c npm run build:labextension && npm run build:nbextension

    npm ERR! A complete log of this run can be found in:
    npm ERR!     /home/rly/.npm/_logs/2021-02-10T01_15_25_445Z-debug.log
    /home/rly/miniconda3/envs/nwb_datajoint/lib/python3.8/site-packages/setuptools/dist.py:458: UserWarning: Normalizing '0.1.0.dev' to '0.1.0.dev0'
      warnings.warn(tmpl.format(**locals()))
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setup.py", line 118, in <module>
        setup(**setup_args)
      File "/home/rly/miniconda3/envs/nwb_datajoint/lib/python3.8/site-packages/setuptools/__init__.py", line 163, in setup
        return distutils.core.setup(**attrs)
      File "/home/rly/miniconda3/envs/nwb_datajoint/lib/python3.8/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/home/rly/miniconda3/envs/nwb_datajoint/lib/python3.8/distutils/dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "/home/rly/miniconda3/envs/nwb_datajoint/lib/python3.8/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setupbase.py", line 474, in run
        [self.run_command(cmd) for cmd in cmds]
      File "/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setupbase.py", line 474, in <listcomp>
        [self.run_command(cmd) for cmd in cmds]
      File "/home/rly/miniconda3/envs/nwb_datajoint/lib/python3.8/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/home/rly/miniconda3/envs/nwb_datajoint/lib/python3.8/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setupbase.py", line 270, in run
        c.run()
      File "/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setupbase.py", line 375, in run
        run(npm_cmd + ['run', build_cmd], cwd=node_package)
      File "/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setupbase.py", line 221, in run
        return subprocess.check_call(cmd, **kwargs)
      File "/home/rly/miniconda3/envs/nwb_datajoint/lib/python3.8/subprocess.py", line 364, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['/home/rly/miniconda3/envs/nwb_datajoint/bin/npm', 'run', 'build:all']' returned non-zero exit status 1.
    ----------------------------------------
ERROR: Command errored out with exit status 1: /home/rly/miniconda3/envs/nwb_datajoint/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setup.py'"'"'; __file__='"'"'/home/rly/labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' develop --no-deps Check the logs for full command output.
```

This occurs both with npm version 6.9.0 and after upgrading npm to version 7.0.14 (`conda install -c conda-forge nodejs`).

Calling `npm run build:all` from the `jupyterlab/labbox_ephys_widgets_jp` directory results in:
```
src/SortingViewPluginComponentWrapper.tsx:3:50 - error TS2307: Cannot find module './extensions/common/getRecordingInfo'.

3 import { useRecordingInfo, useSortingInfo } from './extensions/common/getRecordingInfo';
                                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/pluginWidgets.tsx:15:103 - error TS2724: Module '"./extensions/extensionInterface"' has no exported member 'sortingCurationReducer'. Did you mean 'sortingSelectionReducer'?

15 import { externalUnitMetricsReducer, filterPlugins, Plugins, Recording, RecordingViewPlugin, Sorting, sortingCurationReducer, sortingSelectionReducer, SortingUnitMetricPlugin, SortingUnitViewPlugin, SortingViewPlugin, useRecordingAnimation } from './extensions/extensionInterface';
                                                                                                         ~~~~~~~~~~~~~~~~~~~~~~

  src/extensions/extensionInterface.ts:399:14
    399 export const sortingSelectionReducer: Reducer<SortingSelection, SortingSelectionAction> = (state: SortingSelection, action: SortingSelectionAction): SortingSelection => {
                     ~~~~~~~~~~~~~~~~~~~~~~~
    'sortingSelectionReducer' is declared here.


Found 2 errors.
```

--- a/jupyterlab/labbox_ephys_widgets_jp/src/SortingViewPluginComponentWrapper.tsx
+++ b/jupyterlab/labbox_ephys_widgets_jp/src/SortingViewPluginComponentWrapper.tsx
@@ -1,8 +1,9 @@
 import { WidgetModel } from '@jupyter-widgets/base';
 import React, { FunctionComponent, useEffect, useMemo, useReducer, useRef, useState } from 'react';
-import { useRecordingInfo, useSortingInfo } from './extensions/common/getRecordingInfo';
 import { CalculationPool } from './extensions/common/hither';
 import { AppendOnlyLog, useFeedReducer } from './extensions/common/useFeedReducer';
+import { useRecordingInfo } from './extensions/common/useRecordingInfo';
+import { useSortingInfo } from './extensions/common/useSortingInfo';
 import { sortingCurationReducer, SortingCurationWorkspaceAction } from './extensions/common/workspaceReducer';
 import { externalUnitMetricsReducer, Plugins, Recording, Sorting, SortingCuration, sortingSelectionReducer, SortingViewPlugin, useRecordingAnimation } from './extensions/extensionInterface';
 

--- a/jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx
+++ b/jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx
@@ -12,7 +12,8 @@ import { CalculationPool, createCalculationPool, HitherContext } from './extensi
 import { sleepMsec } from './extensions/common/misc';
 import { useRecordingInfo } from './extensions/common/useRecordingInfo';
 import { useSortingInfo } from './extensions/common/useSortingInfo';
-import { externalUnitMetricsReducer, filterPlugins, Plugins, Recording, RecordingViewPlugin, Sorting, sortingCurationReducer, sortingSelectionReducer, SortingUnitMetricPlugin, SortingUnitViewPlugin, SortingViewPlugin, useRecordingAnimation } from './extensions/extensionInterface';
+import { sortingCurationReducer } from './extensions/common/workspaceReducer';
+import { externalUnitMetricsReducer, filterPlugins, Plugins, Recording, RecordingViewPlugin, Sorting, sortingSelectionReducer, SortingUnitMetricPlugin, SortingUnitViewPlugin, SortingViewPlugin, useRecordingAnimation } from './extensions/extensionInterface';
 import initializeHitherInterface from './extensions/initializeHitherInterface';
 import theme from './extensions/theme';
 import registerExtensions from './registerExtensions';
@@ -192,7 +193,7 @@ const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = (
     model.on('change:curation', () => {
       if (!curationSubfeed) {
         curationDispatch({
-          type: 'SetCuration',
+          type: 'SET_CURATION',
           curation: model.get('curation')
         })
       }

